docker/launch-netbox.sh

```diff
@@ -1,8 +1,33 @@
 #!/bin/bash
 
-UNIT_CONFIG="${UNIT_CONFIG-/etc/unit/nginx-unit.json}"
 UNIT_SOCKET="/opt/unit/unit.sock"
 
+put_config() {
+  RET=$(
+    curl \
+    --silent \
+    --write-out '%{http_code}' \
+    --request PUT \
+    --data-binary "@$1" \
+    --unix-socket $UNIT_SOCKET \
+    http://localhost/$2
+  )
+  RET_BODY=${RET::-3}
+  RET_STATUS=$(echo $RET | tail -c 4)
+
+  if [ "$RET_STATUS" -ne "200" ]; then
+      echo "⚠️ Error: Failed to load configuration from $1"
+      ( echo "HTTP response status code is '$RET_STATUS'"
+        echo "$RET_BODY"
+      ) | sed 's/^/  /'
+
+      kill "$(cat /opt/unit/unit.pid)"
```

Comment:
```suggestion
      kill < /opt/unit/unit.pid
```

I believe the invocation of the subshell and `cat` is not necessary.